app/controllers/application_controller.rb

```diff
@@ -10,6 +10,10 @@ class ApplicationController < ActionController::Base
 
   def redirect_to_setup
     return true unless signed_in?
-    redirect_to setup_path if Minion.assigned_role.count.zero?
+    redirect_to setup_path if no_setup?
+  end
+
+  def no_setup?
```

Comment:
1. Add documentation
2. I like methods to be affirmative (aka without "no_*" and stuff). For example, a better name would be `setup_done?`.